config/initializers/sidekiq.rb

```diff
@@ -4,6 +4,28 @@
 require 'sidekiq/web'
 require './config/environment'
 
+redis_params = {
+  host: APP_CONFIG.redis_host ||= 'localhost',
+  port: APP_CONFIG.redis_port ||= '6379',
```

Comment:
Not really important, but I guess this should be an integer to match the readme.